app/models/reservation.rb

```diff
@@ -1,4 +1,4 @@
 class Reservation < ApplicationRecord
-  belongs_to :user
+  belongs_to :user, optional: true
```

Comment:
Z czego to wynika? Mogą być jakieś rezerwacje bez użytkowników dopisanych? To jest jakby rozwiązanie na godziny, w których boisko jest przez szkołę zajęte?

Reply:
no wlasnie to troche dupowato, powinno byc imho tak ze boiska ma jakies `manager_id` i te rezerwacje byly do tego managera przypisane, czy cos w tym stylu

Reply:
Tak, dokładnie dlatego to dodałem.

Reply:
Hm, okay przemyślę to zaraz jeszcze.